GameServer/realmabilities/handlers/rr5/AssassinateAbility.cs

```diff
@@ -0,0 +1,75 @@
+using DOL.Database;
+using DOL.GS.PacketHandler;
+using DOL.GS.Spells;
+using DOL.Language;
+
+namespace DOL.GS.RealmAbilities
+{
+    /// <summary>
+    /// Assassinate inherit of RR5RealmAbility (RealmLevel >= 40 and free)
+    /// </summary>
+    public class AssassinateAbility : RR5RealmAbility
```

Comment:
This is also not an RR5 ability anymore.

Reply:
Fixed, now it's a TimedRealmAbility